app/controllers/application_controller.rb

```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+  helper_method :current_user
+
+  def current_user
+    @current_user ||= User.first
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/forms_project_v1_1.md#:~:text=It%20will%20return%20the%20first%20user%20from%20the%20database.), The `current_user` method suppose to return the first user only. Please fix that. 
```suggestion

  def current_user
    User.first
  end
```